### DISCUSSION TITLE
Subject Wise Ranking Needed

### DISCUSSION THREAD

**Discussion by arjunsuresh** [GATEOverflow/GATE-Marks-Evaluator#3] [Ideas]
We need to add subject wise details so that we can have Pragy's app display subject wise ranking. For this we need the following enhancement to the app.

A subject selection (multi-select) box which should populate its value from a JSON file (subject name, subject ID pairs). That's all required for working. Main task is to do this in a neat way maintaining the look and feel of the app. Please submit this as a pull request. Best submission gets a reward and will be merged to the live App. :)

The modifications should be in https://github.com/GATEOverflow/GATE-Marks-Evaluator/blob/master/VisualizeMarks.php

**Comment by smn98** [comment]
Is the backend work already done?

**Comment by Default2882** [comment]
Hello, is there an example JSON file which has the (subject name, subject ID) pair, so I can take a look at it?

**Comment by mukeshgupta7** [comment]
Can you specify the url for requesting the server which return JSON response.
